### PR TITLE
Buy Anything Button subpage

### PR DESCRIPTION
**Pull request by CAnorbo**
Add Buy Anything Button subpage to the website.

### CODE REVIEW

File: wwwroot/buy-anything.html [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+                                <a href="index.html#download"><img src="img/bab.svg" class="w-50 my-3" />
+                                </a>
+                            </div>
+                        </div>
+                        <div class="col-md-6 offset-md-1">
+                            <div class="row my-5">
+                                <div class="col-12 mb-4">
+                                    <div class="feature-box">
+                                        <div class="title my-3 green-text">1. Your Personal Concierge</div>
+                                        <p>Our feature connects you with a personal concierge from ShopinBit, ensuring a private and secure shopping experience.</p>
```
Comment (MaxHillebrand) [inline wwwroot/buy-anything.html:109]:
```suggestion
                                        <p>Our feature connects you with a personal team concierge agents from ShopinBit, ensuring a private and secure shopping experience.</p>
```

File: wwwroot/buy-anything.html [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+                </div>
+            </section>
+            <section class="p-4 overflow">
+                <div class="container mb-5">
+                    <div class="row my-5">
+                        <div class="col-md-5">
+                            <div class="sticky-top">
+                                <h2 class="pt-header title-lined left">Effortless Bitcoin Purchases</h2>
+
+                                <p>With the Wasabi Wallet "Buy Anything with Bitcoin" feature, you can purchase flights, cars, or even ASIC miners with ease, all while maintaining your privacy and from the comfort of your wallet.</p>
```
Comment (MaxHillebrand) [inline wwwroot/buy-anything.html:99]:
```suggestion
                                <p>With the Wasabi Wallet "Buy Anything with Bitcoin" feature, you can purchase flights, ASIC miners, or even cars with ease, all while maintaining your privacy and from the comfort of your wallet.</p>
```

**Comment by CAnorbo** [comment]
> Concept ACK. But guess the plan is to publish this together with release, right?

Exactly.

File: wwwroot/buy-anything.html [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
+
+                </div>
+            </section>
+            <section class="home p-4 green-bg">
+                <div class="container mb-5">
+                    <div class="row align-items-center">
+                        <div class="col-12 col-md-6">
+                            <div class="my-5">
+                                <h2 class="pt-header title-lined left">Exclusive Rewards</h2>
+                                <p class="big-text">Don't miss out on our exclusive offer! Random buyers using the new feature stand a chance to win <strong>$100 vouchers</strong>. Act fast, offer ends December 14th XX:XX UTC.</p>
```
Comment (MaxHillebrand) [inline wwwroot/buy-anything.html:137]: This isn't ever green content, remember to remove it later.

File: wwwroot/about.html [hunk 4]
```diff
@@ -45,9 +45,6 @@
                     <li class="nav-item">
                         <a class="nav-link" href="index.html">The Wallet</a>
                     </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="contribution.html">Contribution</a>
-                    </li>
```
Comment (kristapsk) [inline wwwroot/about.html:50]: Why this change?

File: wwwroot/buy-anything.html [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+                                    <p>It will depend on the source and destination of the product. If it has to cross borders, it might be subject to duties or taxes. If it can be purchased in the same country as it’s being delivered, there will be no duty taxes added to the final price.</p>
+                                </div>
+                            </div>
+                            <div class="card">
+                                <input type="checkbox" id="faq7">
+                                <label for="faq7">
+                                    7. How long does it take to ship a physical product? A digital product?
+                                </label>
+                                <div class="card-content">
+                                    <p>YThere is no general answer to this question for physical products. It depends on where the product is (ShopinBit’s warehouse or third party warehouse) and how it will be shipped to you.
```
Comment (MaxHillebrand) [inline wwwroot/buy-anything.html:225]:
```suggestion
                                    <p>There is no general answer to this question for physical products. It depends on where the product is (ShopinBit’s warehouse or third party warehouse) and how it will be shipped to you.
```

File: wwwroot/buy-anything.html [hunk 6]
```diff
@@ -0,0 +1,414 @@
...
+                                        <br> d) It is all legal to do so.
+                                        <br>
+                                        <br> An offer will be issued only until these 4 points are sorted out. If it’s not possible, you will receive a friendly decline with an explanation of why.
+                                    </p>
+                                </div>
+                            </div>
+                            <div class="card">
+                                <input type="checkbox" id="faq4">
+                                <label for="faq4">
+                                    4. How is the bitcoin price calculated at the time of the transaction?
```
Comment (kristapsk) [inline wwwroot/buy-anything.html:195]: Shouldn't it be "Bitcoin", not "bitcoin", in this context?
```suggestion
                                    4. How is the Bitcoin price calculated at the time of the transaction?
```

File: wwwroot/buy-anything.html (hunk 5 above)
Comment (kristapsk) [inline wwwroot/buy-anything.html:225]:
```suggestion
                                    <p>There is no general answer to this question for physical products. It depends on where the product is (ShopinBit’s warehouse or third party warehouse) and how it will be shipped to you.
```

**Review by MaxHillebrand — CHANGES_REQUESTED** [review]

File: wwwroot/buy-anything.html [hunk 7]
```diff
@@ -0,0 +1,414 @@
...
+                    </div>
+                    <div class="row mb-5">
+                        <div class="col-12">
+                            <div class="card">
+                                <input type="checkbox" id="faq1">
+                                <label for="faq1">
+                                    1. Can I really buy anything?
+                                </label>
+                                <div class="card-content">
+                                    <p>Shopinbit's Concierge Service does not provide any kind of drugs, prescription medications, weapons, precious metals, cryptocurrencies, financial instruments, or any type of product that is illegal in Poland. You can buy anything else.</p>
```
Comment (kristapsk) [inline wwwroot/buy-anything.html:163]: It's at some places "ShopinBit", at others "Shopinbit", should be consistent everywhere.

File: wwwroot/buy-anything.html [hunk 8]
```diff
@@ -0,0 +1,414 @@
...
+                                    </p>
+                                </div>
+                            </div>
+                            <div class="card">
+                                <input type="checkbox" id="faq13">
+                                <label for="faq13">
+                                    13. What is the return and reimbursement policy?
+                                </label>
+                                <div class="card-content">
+                                    <p>After you receive your order, you have a 14-day right of return. This means that if you do not like it, you can send it back (you will have to pay for the shipping).</p>
```
Comment (MarnixCroes) [inline wwwroot/buy-anything.html:288]: I'd mention something like: please refer to Shopinbit's Terms and Conditions for up-to-date conditions.
We cannot be sure it is always up to date/Shopinbit didn't make any changes in the meantime

File: wwwroot/about.html (hunk 4 above)
Comment (CAnorbo) [reply wwwroot/about.html:50]: No space for more buttons at the header. Contribution moved to the footer.

File: wwwroot/buy-anything.html (hunk 7 above)
Comment (MarnixCroes) [reply wwwroot/buy-anything.html]: *should be _Shopinbit_ everywhere, that's what they use themselves

File: wwwroot/buy-anything.html (hunk 8 above)
Comment (kristapsk) [reply wwwroot/buy-anything.html:286]: 14 day return policy is EU law, not their choice, so will not change. And if it will, we will notice.

File: wwwroot/buy-anything.html [hunk 9]
```diff
@@ -0,0 +1,414 @@
...
+                                </label>
+                                <div class="card-content">
+                                    <p>All requests are handled manually by ShopinBit's team and sometimes need to be reviewed with legal counsel. The following 4 points have to be reviewed:
+                                        <br>
+                                        <br> a) That the product/service can be bought from ShopinBit;
+                                        <br> b) ShopinBit is allowed to sell the product to you;
+                                        <br> c) ShopinBit can ship the product to you;
+                                        <br> d) It is all legal to do so.
+                                        <br>
+                                        <br> An offer will be issued only until these 4 points are sorted out. If it’s not possible, you will receive a friendly decline with an explanation of why.
```
Comment (MarnixCroes) [inline wwwroot/buy-anything.html:186]:
```suggestion
                                        <br> An offer will only be made if these 4 points are resolved. If it’s not possible, you will receive a friendly decline with an explanation.
```

**Review by MarnixCroes — COMMENTED** [review]
bab.png is unused?
_ShopinBit_ should be changed to _Shopinbit_. That's their official name

**Comment by CAnorbo** [comment]
> bab.png is unused? _ShopinBit_ should be changed to _Shopinbit_. That's their official name

They are using ShopinBit on their website. I also asked Lando and he said ShopinBit is correct.
"It's ShopinBit (Shopping in Bitcoin)"

**Comment by MarnixCroes** [comment]
> > bab.png is unused? _ShopinBit_ should be changed to _Shopinbit_. That's their official name
> 
> They are using ShopinBit on their website. I also asked Lando and he said ShopinBit is correct. "It's ShopinBit (Shopping in Bitcoin)"

haha ok sry.
I was reading their T&C and there they use lower case everywhere. in other part of website it's uppercase I see

File: wwwroot/buy-anything.html [hunk 10]
```diff
@@ -0,0 +1,414 @@
...
+                                    </p>
+                                </div>
+                            </div>
+                            <div class="card">
+                                <input type="checkbox" id="faq11">
+                                <label for="faq11">
+                                    11. In which countries is this available?
+                                </label>
+                                <div class="card-content">
+                                    <p>This service is available for all countries, except Iran and North Korea. And currently due to the current conditions, Ukraine isn't available either.
```
Comment (MarnixCroes) [inline wwwroot/buy-anything.html:266]:
```suggestion
                                    <p>This service is available for all countries, except Iran, North Korea, Russia and Ukraine.
```

Comment (MarnixCroes) [reply wwwroot/buy-anything.html:266]: Russia [isn't available either](https://shopinbit.com/wasabiwelcome).

File: wwwroot/buy-anything.html (hunk 3 above)
Comment (MarnixCroes) [reply wwwroot/buy-anything.html:135]: date is already over. remove or update it

**Comment by MarnixCroes** [comment]
@CAnorbo 
is this PR ready? i.e. just waiting for merge?

should it still be Christmas focused?
if release is tomorrow or day after, than the page needs to be updated again in a couple days

File: wwwroot/buy-anything.html (hunk 3 above)
Comment (kristapsk) [reply wwwroot/buy-anything.html:135]: Reminder that this needs to be changed or removed.

**Review by MaxHillebrand — APPROVED** [review]

**Comment by MarnixCroes** [comment]
image

https://github.com/zkSNACKs/WasabiWalletWebSite/pull/21#discussion_r1430698494
